code/game/objects/items/oddities.dm

```diff
@@ -648,3 +667,137 @@
 				to_chat(user, "You fail to solve the equation, did you carry the [rand(1, 9)]?")
 	else
 		to_chat(user, "You're not smart enough to comprehend what this says.")
+
+
+//May one day I be forgiven
+/obj/item/oddity/common/justificande
+	name = "seven-sided coin"
+	desc = "An odd seven-sided coin with somewhat sharp edges and blank sides. You have a strange urge to flip it."
+	icon_state = "justificande"
+	prob_perk = 0 //No perk for the sake of balance
+	oddity_stats = list(
+		STAT_COG = 3,
+		STAT_VIG = 2,
+		STAT_ROB = 2,
+		STAT_TGH = 2,
+		STAT_BIO = 3,
+		STAT_MEC = 3
+	)
+
+	var/heads_stats = list(
+
+		STAT_VIG = 5,
+		STAT_ROB = 5,
+		STAT_TGH = 5
+	)
+
+	var/tails_stats = list(
+		STAT_COG = 5,
+		STAT_BIO = 5,
+		STAT_MEC = 5
+	)
+
+	//It is, after all, the Number.
+	var/sideways_stats = list(
+		STAT_COG = 7,
+		STAT_BIO = 7,
+		STAT_MEC = 7,
+		STAT_VIG = 7,
+		STAT_ROB = 7,
+		STAT_TGH = 7
+	)
+
+	var/was_flipped = FALSE
+
+
+
+/obj/item/oddity/common/justificande/attack_self(mob/living/carbon/human/user)
+	if(istype(user))
+		if(alert(user, "Flip the coin?", "seven-sided coin", "Yes", "No") == "Yes")
+			if(prob(1))
+				user.visible_message(SPAN_NOTICE("[user] has thrown \the [src]. It lands on its side?! "), \
+						 SPAN_NOTICE("You throw \the [src]. It lands on its side?! "))
+				if(!was_flipped)
+					oddity_stats = sideways_stats
+					desc = "An odd seven-sided coin with somewhat sharp edges and blank sides. Holding it gives you a strange sense of nonspecific closure."
+			else
+				var/result = rand(1, 2)
+				var/comment = ""
+				if(result == 1)
+					comment = "tails"
+					if(!was_flipped)
+						oddity_stats = tails_stats
+						desc = "An odd seven-sided coin with somewhat sharp edges. ONE DAY YOU WILL FORGIVE is stamped on the front."
+				else if(result == 2)
+					if(!was_flipped)
+						oddity_stats = heads_stats
+						desc = "An odd seven-sided coin with somewhat sharp edges. ONE DAY YOU WILL FORGIVE is stamped on the back."
+					comment = "heads"
+
+				user.visible_message(SPAN_NOTICE("[user] has thrown \the [src]. It lands on [comment]! "), \
+									 SPAN_NOTICE("You throw \the [src]. It lands on [comment]! "))
+			was_flipped = TRUE
+			var/datum/component/inspiration/odd = GetComponent(/datum/component/inspiration)
+			odd.stats = oddity_stats
+
+		else
+			to_chat(user, "You have decided against flipping the coin.")
+
+//Episode 8, coming never
+/obj/item/oddity/common/anomazon_card
+	name = "expired anomazon dismembership card"
+	desc = "A gaudy-looking plastic card stamped with a slogan - \"All good things cost an arm and a leg!\" You have a bad feeling about this."
+	icon_state = "anomazon"
+	oddity_stats = list(
+		STAT_ROB = 7,
+		STAT_TGH = 7,
+		STAT_VIG = 7,
```

Comment:
```suggestion
		STAT_VIG = 7
```

Reply:
It's fun, quite a bit of oddities have that trailing comma